.github/workflows/release.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           token: ${{ secrets.DESIGN_REPO_ACCESS_TOKEN }}
           repository: mongodb/design
           event-type: release-leafygreen-ui
-          client-payload: '{"ref": "${{ github.ref }}", "sha": "${{ github.sha }}"", "packages": "${{steps.changesets.outputs.publishedPackages}}"}'
+          client-payload: '{"ref": "${{ github.ref }}", "sha": "${{ github.sha }}"", "packages": \'${{steps.changesets.outputs.publishedPackages}}\'}'
```

Comment:
Why would this not work but `"${{ github.ref }}"` does?

Reply:
Looks like the actual issue is the extra double-quote after `"${{ github.sha }}""
`

Reply:
You're so right. Thanks for catching this!